Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,72 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render a `<picture>` element for responsive image support.
+public struct Picture: Component {
+
+    /// Image to use as fallback.
+    public var image: Image
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public var sources: [Source]
+
+    /// Initialize a `Picture` component with multiple `source` elements.
+    internal init(image: Image, sources: [Picture.Source] = []) {
+        self.image = image
+        self.sources = sources
+    }
+
+    /// Initialize a `Picture` component with a single `source` element.
+    /// Image to use as fallback.
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public init(image: Plot.Image, source: Source) {
+        self.init(image: image, sources: [source])
+    }
+
+    @ComponentBuilder
+    public var body: Component {
+//        if let sources = sources {
```

Comment:
Remove all this commented-out code, please (and you can also remove the comment within the `forEach` as well, I think, as the code is quite self-explanatory).

Reply:
Sorry, thought I had done that already. :-)